ui/packages/shared/profile/src/MetricsGraph/MetricsTooltip/index.tsx

```diff
@@ -121,7 +121,7 @@ const MetricsTooltip = ({
                       <tr>
                         <td className="w-1/4">Value</td>
                         <td className="w-3/4">
-                          {valueFormatter(highlighted.valuePerSecond, sampleUnit, 5)}
+                          {valueFormatter(highlighted.valuePerSecond, sampleUnit, 5)}{' '}
```

Comment:
I don't believe the space is necessary here.

Reply:
Good catch! That's a left over of something I removed again.